modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts

```diff
@@ -258,16 +280,17 @@ export default class Tiles3DConverter {
     } else {
       const nodeUrl = this._relativeUrlToFullUrl(parentNode.url, childNodeInfo.href!);
       // load metadata
-      const options = {
+      const options: I3SLoaderOptions = {
         i3s: {
           ...this.loaderOptions,
+          // @ts-expect-error
           isTileHeader: true,
           loadContent: false
         }
       };
 
       console.log(`Node conversion: ${nodeUrl}`); // eslint-disable-line no-console,no-undef
-      header = await load(nodeUrl, I3SLoader, options);
+      header = await loadFromArchive(nodeUrl, I3SLoader, options, this.slpkFilesystem);
```

Comment:
Ideally this would generalize this to `loadFromFilesystem` and we would make the archive class follow a file system pattern

Reply:
`loadFromArchive` is a bad name, probably . It is a function that decides, is the URL an archive or not. It creates a Filesystem if it is an archive, and loads directly if it isn't.